src/gui/qgsrubberband.h

```diff
@@ -77,49 +77,57 @@ class GUI_EXPORT QgsRubberBand: public QgsMapCanvasItem
     QgsRubberBand( QgsMapCanvas *mapCanvas, QgsWkbTypes::GeometryType geometryType = QgsWkbTypes::LineGeometry );
 
     /**
-     * Set the color for the rubberband
+     * Sets the color for the rubberband
      *  @param color  The color used to render this rubberband
      */
     void setColor( const QColor &color );
 
     /**
-     * Set the fill color for the rubberband
+     * Sets the fill color for the rubberband
      *  @param color  The color used to render this rubberband
      *  @note Added in 2.6
      */
     void setFillColor( const QColor &color );
 
     /**
-     * Set the stroke color for the rubberband
+     * Sets the stroke color for the rubberband
      *  @param color  The color used to render this rubberband
      *  @note Added in 2.6
      */
     void setStrokeColor( const QColor &color );
 
     /**
-     * Set the width of the line. Stroke width for polygon.
+     * Sets a secondary stroke color for the rubberband which will be
+     * drawn under the main stroke color
+     *  @param color  The color used to render a secondary stroke color to this rubberband
```

Comment:
Can you also add a getter? It's pretty frustrating when you hit the time that you need to retrieve the existing value and it's just not possible